dex/testing/firo/electrumx.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+#
+# Set up ElectrumX-Firo (electrumX) regtest server for testing.
+# - Expects the firo regtest chain harness to be running and listening 
```

Comment:
nit: There are some "bad spaces" in this file, or spaces at the end of lines that aren't needed. For example at the end of `listening` here. Your editor may have a setting to show these better.

Reply:
My editor is a `vscode` extension and not good for MD. I will lookat these later for sure in a couple of editors.

Reply:
Done